plugin/trino-hive/src/main/java/io/trino/plugin/hive/parquet/ParquetWriterConfig.java

```diff
@@ -63,9 +67,9 @@ public boolean isParquetOptimizedWriterEnabled()
         return parquetOptimizedWriterEnabled;
     }
 
-    @Config("parquet.experimental-optimized-writer.enabled")
-    @LegacyConfig("hive.parquet.optimized-writer.enabled")
-    @ConfigDescription("Experimental: Enable optimized Parquet writer")
+    @Config("parquet.optimized-writer.enabled")
+    @LegacyConfig("parquet.experimental-optimized-writer.enabled")
```

Comment:
`hive.parquet.optimized-writer.enabled`  is no longer supported?
Why not?

Reply:
I thought to remove it since it is already marked as legacy, but let me keep it for now as it's not necessary to remove.

Reply:
Turns out that using multiple LegacyConfig runs into the problem that `assertDeprecatedEquivalence` doesn't allow testing this case and complains about untested property. I think I'd have to remove `TestParquetWriterConfig#testLegacyProperties` if I keep multiple LegacyConfig.
WDYT ?

Reply:
What's the problem exactly? There are other config classes with multiple legacy configs: https://github.com/trinodb/trino/blob/master/core/trino-main/src/main/java/io/trino/execution/scheduler/NodeSchedulerConfig.java#L112

Reply:
>  If we have marked something deprecated, it seems that making it defunct after some time should be a valid change.

Yeah, that seems reasonable to me. It has been two years

Reply:
> Turns out that using multiple LegacyConfig runs into the problem that `assertDeprecatedEquivalence` doesn't allow testing this case and complains about untested property

it does allow testing this case, but quite awkwardly.
i pushed a fix in this PR

Reply:
Nice, thanks :)